README.md

```diff
@@ -660,6 +660,74 @@ angular.module('AlertListComponentModule', [])
 - Don’t manipulate DOM in your controllers, this will make them harder to test. Use directives instead.
 
 
+### Services and Factories
+
+#### Services should always export an object, not just a single function
```

Comment:
Could maybe be re-worded, not sure exactly to what. `Treat Service objects like a Class with static methods; don't export services as a single function`